src/components/DateField/Example.md

```diff
@@ -1,95 +1,476 @@
 **Elementary**
 
-The `DateField` component provides an easy way for selecting a date. As all inputs it can be used in a controlled and managed mode.
+The `DateField` component provides an easy way for selecting a date. As all inputs it can be used in a controlled and managed mode. 
+
+`DateField` is based on [ReactJS Datepicker](https://reactdatepicker.com/). Please refer to ReactJS Datepicker Docs for the detailed props description.
 
 ```jsx
 const { DateField } = require('precise-ui');
 
 <DateField onChange={(e) => console.log(e)} />
 ```
 
-**Customization**
+`DateField` with `autoComplete`
 
-Abbreviations may be given to support localization and customization.
+```jsx { "props": { "data-skip": true } }
+const { DateField } = require('precise-ui');
 
-```jsx
+<DateField onChange={(e) => console.log(e)} autoComplete />
+```
+
+`DateField` with custom date format
+
+```jsx { "props": { "data-skip": true } }
+const { DateField } = require('precise-ui');
+
+<DateField onChange={(e) => console.log(e)} dateFormat="yyyy/mm/dd" />
+```
+
+Select time
+
+```jsx { "props": { "data-skip": true } }
+const { DateField } = require('precise-ui');
+
+<DateField showTimeSelect timeFormat="HH:mm" timeIntervals={15} dateFormat="MMMM d, yyyy h:mm aa" onChange={(e) => console.log(e)}/>
+```
+
+Select time only
+
+```jsx { "props": { "data-skip": true } }
 const { DateField } = require('precise-ui');
 
 <DateField
-  defaultValue="2018-01-08"
+  showTimeSelect
+  showTimeSelectOnly
+  timeIntervals={15}
+  dateFormat="h:mm aa"
+  timeCaption="Time"
   onChange={(e) => console.log(e)}
-  months={['J', 'F', 'M', 'A', 'M', 'J', 'J', 'A', 'S', 'O', 'N', 'D']}
-  weekDays={['Mo', 'Di', 'Mi', 'Do', 'Fr', 'Sa', 'So']}  />
+/>
 ```
 
-**Controlled Mode**
+Exclude Times
 
-The calendar open state can be controlled by using the `open` prop.
+```jsx { "props": { "data-skip": true } }
+const { DateField } = require('precise-ui');
+const { setHours, setMinutes } = require('date-fns');
 
-```jsx
-const { DateField, Button } = require('precise-ui');
-
-class ToggleCalendar extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-        open: false,
-    };
-    this.click = () => {
-      this.setState({
-          open: !this.state.open,
-      });
-    };
-  }
-
-  render() {
-    return (
-      <>
-        <Button onClick={this.click}>toggle</Button>
-        <DateField open={this.state.open} />
-      </>
-    );
-  }
+<DateField
+  showTimeSelect
+  excludeTimes={[setHours(setMinutes(new Date(), 0), 17), setHours(setMinutes(new Date(), 30), 18), setHours(setMinutes(new Date(), 30), 19), setHours(setMinutes(new Date(), 30), 17)]}
+  dateFormat="MMMM d, yyyy h:mm aa"
+  onChange={(e) => console.log(e)}
+/>
+```
+
+Include Times
+
+```jsx { "props": { "data-skip": true } }
```

Comment:
why do you skip most of the examples?

Reply:
there is no value to visual test them as on the screenshot it will be just a TextField, but testing takes time. I don't skip all `inline` examples because they allow to actually test the appearance of the component